resource-managers/yarn/src/main/scala/org/apache/spark/deploy/yarn/Client.scala

```diff
@@ -17,7 +17,7 @@
 
 package org.apache.spark.deploy.yarn
 
-import java.io.{FileSystem => _, _}
+import java.io.{File, FileOutputStream, IOException, OutputStreamWriter, _}
```

Comment:
You don't need to change this line. The previous one was more correct.

Reply:
Fixed